src/commands/resume.js

```diff
@@ -16,0 +1,15 @@
+const Player = require('../audio/player.js');
+
+module.exports = {
+    name: 'resume',
+    run: function(event) {
+        let player;
+        if (!Player.getSendingPlayer(event.guild)) {
+            return event.channel.send('❌ 必須要有音樂正在播放');
+        } else {
+            player = Player.getSendingPlayer(event.guild);

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
            player = Player.getSendingPlayer(event.guild);
```